Symfony/src/AppBundle/Controller/RegisterController.php

```diff
@@ -20,9 +20,9 @@ public function registerAction(Request $request)
         if ($form->isSubmitted() && $form->isValid()) {
             $password = $this->get('security.password_encoder')->encodePassword($user, $user->getPassword());
             $user->setPassword($password)->setSuccessfulFlashCount(0)->setFailedFlashCount(0)->setTotalFlashCount(0)->setTier(0);
-            $em = $this->getDoctrine()->getManager();
-            $em->persist($user);
-            $em->flush();
+            $doc = $this->getDoctrine()->getManager();
```

Comment:
So now we went from a cryptic `$em` to another cryptic `$doc`. Does it mean `document`?

The idea behind the "no variable should be shorter than 3 letters" is to attempt to enforce variable names that have a name that is descriptive of their value and noone can misunderstand what they do. So with this change you've circumvented the rule to "cheat" the tool but the initial reason it was complaining about remained (if not become worse). Changing this to `$entityManager` (which `em` stands for) would have been an acceptable solution.